src/django_idom/websocket_consumer.py

```diff
@@ -15,14 +18,37 @@
 _logger = logging.getLogger(__name__)
 
 
-class IdomAsyncWebSocketConsumer(AsyncJsonWebsocketConsumer):
+@dataclass
+class WebsocketConnection:
+    scope: dict
+    close: Callable[[Optional[int]], Coroutine[Any, Any, None]]
```

Comment:
You should be able to just replace `Coroutine[Any, Any, None]` with `Awaitable[None]`

Reply:
Alright, completed.